EventListener/UserContextSubscriber.php

```diff
@@ -61,21 +62,28 @@ class UserContextSubscriber implements EventSubscriberInterface
      */
     private $ttl;
 
+    /**
```

Comment:
actually, its not only performance. if we would boot the kernel, we would treat the expired session as anonymous which is exactly what #274 wanted to prevent. maybe explain the thing about anonymous fake hashes in the phpdoc here so we still know why this is done like this.